update_crawler.py

```diff
@@ -59,13 +90,21 @@ def get_source_packages(self, project, expand=False):
     def _get_source_package(self, project, package):
         return http_GET(makeurl(self.apiurl,
                                 ['source', project, package],
-                                {'view': 'info'})).read()
+                                {
+                                    'view': 'info',
+                                    'parse': 1,
+                                })).read()
 
     def get_source_verifymd5(self, project, package):
         """ Return the verifymd5 of a source package."""
         root = ET.fromstring(self._get_source_package(project, package))
         return root.get('verifymd5')
 
+    def get_source_version(self, project, package):
+        """ Return the version of a source package."""
+        root = ET.fromstring(self._get_source_package(project, package))
+        return stringToVersion(root.find('version').text)
```

Comment:
are you sure this is needed? Looks like obs returns version and release in separate tags